source/_patterns/04-pages/home.json

```diff
@@ -377,8 +377,7 @@
               }
             },
             "formats": {
-              "html": true,
-              "pdf": true
+              "list": ["Executable", "PDF"]
```

Comment:
👋

Reply:
shouldn't this also have `HTML`?

Reply:
I know this is just an example, but is the uppercase implemented via CSS?

Reply:
> shouldn't this also have HTML?

Ah re-read the whole issue, there's... not enough space for that? Whatever, the example is intended.